acme/acme/challenges_test.py

```diff
@@ -264,7 +264,9 @@ def test_gen_verify_cert_gen_key(self):
 
     def test_verify_bad_cert(self):
         self.assertFalse(self.response.verify_cert(
-            test_util.load_cert('cert.pem')))
+            OpenSSL.crypto.load_certificate(
```

Comment:
Alternatively, we could just check in with Debian, Redhat and other folks making backport packages to ensure they run full tox tests on the things they build before uploading them to backports.  I believe that is already true for debian but @hlieberman should confirm.